packages/md-extractor/.env.example

```
@@ -5,3 +5,9 @@ API_SECRET_KEY={your_api_key} # openssl rand -base64 32
 SERVER_HOST="0.0.0.0"
 SERVER_PORT=8080
 HOT_RELOAD=False
+LOG_LEVEL=INFO
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_LEVEL key should go before the SERVER_HOST key

(UnorderedKey)